apps/vscode/extension/src/WebViewMessageHandler.ts

```diff
@@ -74,24 +73,11 @@ export class WebViewMessageHandler {
 			}
 			case 'vscode:bookmark/request': {
 				const url = e.data.url
-				fetch(BOOKMARK_ENDPOINT, {
-					method: 'POST',
-					headers: {
-						'Content-Type': 'application/json',
-						// We can fake the origin here because we're in node.js
-						origin: 'https://www.tldraw.com',
-					},
-					body: JSON.stringify({
-						url,
-					}),
-				})
-					.then((resp) => {
-						return resp.json()
-					})
-					.then((json: any) => {
+				await unfurl(url)
+					.then(async (json: any) => {
 						this.webviewPanel.webview.postMessage({
 							type: 'vscode:bookmark/response',
-							uuid: e.uuid,
+							uuid: e.uuid + '_response',
```

Comment:
add a reference to the comment here, otherwise it's unclear why this `_response` is here :)